CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
+Unreleased
+-----------------------------
+* Use Tutor v1 plugin API
+
+
 Version 0.2.0 (2022-06-28)
 -----------------------------
 * Add `HASTEXO_ENABLE_SUSPENDER` and `HASTEXO_ENABLE_REAPER` to
   effectively enable/disable the `suspender` and `reaper` deployments
   (by setting their respective replica counts to 1 or 0).
+* Add options to enable/disable the `suspender` and
```

Comment:
This bullet point is superfluous. Probably slipped through during a rebase on current `main`.

Reply:
Yes, I'll fix